acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -64,6 +64,15 @@
   sitepp_content = generate_manifest(test_resources)
   assertion_code = generate_assertions(test_resources)
 
+  refutation_resources = test_resources.collect do |assertion_group|
```

Comment:
> thus was the "elegance" in the previous solution. you could just run the assertions in each state (debug output, syslog, etc) and they would always and should always be the same

I think the only reason the syslog ones passed before is that the agent run console debug output was captured in the Beaker logger and the syslog assertions were looking at the Beaker logger - and not strictly the content of the `syslog` file.